packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -156,46 +161,72 @@ abstract class InspectorControllerClient {
 }
 
 class InspectorTree extends StatefulWidget {
-  const InspectorTree({Key key, this.controller}) : super(key: key);
+  const InspectorTree({
+    Key key,
+    this.controller,
+    this.isSummaryTree = false,
+  }) : super(key: key);
 
-  final InspectorTreeControllerFlutter controller;
+  final InspectorController controller;
```

Comment:
why does the inspectorTree need a reference to the InspectorController instead of the InspectorTreeController? I think you should be able to switch this back to the InspectorTreeControllerFlutter.

Reply:
I need to pass down the InspectorController to the inspectorTree (particularly the summary tree one) to listen for `debugSummaryLayoutEnabled` value changes. Since we can get in the InspectorTreeController from InspectorController, I didn't include additional field but instead changed it this way

Reply:
One alternative solution is to move back the `debugSummaryLayoutEnabled` to `InspectorTreeControllerFlutter` but it's a bit weird since the field will only be used in the summaryTree one (it will be useless in the detailsTree controller)

Reply:
One other alternative is to make the `debugSummaryLayoutEnabled` as a static attribute.
What do you think?

Reply:
ended up passing the ValueNotifier object directly